docs/TESTING_FIXTURES.md

```
@@ -0,0 +1,110 @@
+```
+shared/presentation/src/androidUnitTest/kotlin/.../common/test_utils/
+  compose/     BisqComposeUiTestBase, PresentationKoinComposeTestBase, …
+  coroutines/  PresentationKoinTestBase, PlatformPresentationKoinTestBase
+  di/          presentationTestModule(...)
+```
+
+`:shared:presentation-test-utils` was removed.
+
+`clientApp` reuses helpers via `kotlin.srcDirs` grafts in `apps/clientApp/build.gradle.kts` — not a module dependency, so no cycle. Point only at `compose/`, `coroutines/`, and `di/` (not the `test_utils` root). Do **not** use `kotlin.include` on that source set: it replaces the default `**/*` and drops clientApp's own `androidUnitTest` sources.
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)